pkgs/development/tools/ginkgo/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, go, buildGoModule, fetchFromGitHub }:
```

One nit, otherwise lgtm.

```suggestion
{ stdenv, buildGoModule, fetchFromGitHub }:
```

---

Fixed :+1: